### PR TITLE
fix msp api-version handling

### PR DESCRIPTION
**Pull request by bkleiner**

### CODE REVIEW

File: src/serial/msp.ts [hunk 1]
```diff
@@ -210,14 +212,12 @@ export class MSPPassthrough {
 
   private async enable() {
     const version = await this.msp.send(MSPCmd.MSP_API_VERSION);
-    if (version.payload[1] < 1 || version.payload[2] < 42) {
+
+    const mspVersion = version.payload.map((n) => "" + n).join(".");
+    Log.debug("msp", "api version", mspVersion);
+    if (!semverSatisfies(mspVersion, MSP_MIN_VERSION)) {
```
Comment (JyeSmith) [inline src/serial/msp.ts:218]: Should be `semver.satisfies` ?

**Review by JyeSmith — APPROVED** [review]

File: src/serial/msp.ts [hunk 2]
```diff
@@ -210,14 +212,12 @@ export class MSPPassthrough {
...
       throw new Error("unsupported msp version");
```
Comment (JyeSmith) [inline src/serial/msp.ts]: Let also add mspVersion to the error, similar to the other errors below.

Comment (bkleiner) [reply src/serial/msp.ts]: done

**Comment by JyeSmith** [comment]
LGTM, released and @pitts-mo testing!

**Comment by bkleiner** [comment]
putting together a test setup myself right now, will report/tack on more changes if necessary

**Comment by bkleiner** [comment]
jup that seems to have done it!

**Pull request merged**